Generate Quote/quotes resource

rjdellecese opened the pull request.

Commit:
Generate Quote/quotes resource

MrMonsk commented on web/router.ex:
```diff
@@ -17,6 +17,7 @@ defmodule HonGraffitiPhoenix.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
+    resources "/quotes", QuoteController
```
We should make this the root URL

rjdellecese replied:
Truth! Will update.

MrMonsk reviewed (approved):
Overall looks good, maybe some naming stuff but I like being flexible with naming

Commit:
Make Quote root for now

Keeping the PageController around because I know we're building an About
page in it right now. Quote will likely not remain the root.

Commit:
Remove Page index layout for now
